extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -147,11 +151,14 @@ export class LiveListManager {
   fetchDocument_() {
     let url = this.url_;
     if (this.latestUpdateTime_ > 0) {
-      url = addParamToUrl(
-        url,
-        'amp_latest_update_time',
-        String(this.latestUpdateTime_)
-      );
+      url = addParamsToUrl(url, {
+        'amp_latest_update_time': String(this.latestUpdateTime_),
```

Comment:
@erwinmombay Wasn't this param meant for cache busting?

Reply:
This doesn't help, because it doesn't change after the client learned about a new update.